src/styles/blocks/content.scss

```diff
@@ -0,0 +1,109 @@
+.container {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  margin: auto;
+  width: 1024px;
+  padding: 0 40px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  padding: 29px 14px 14px;
+  margin-bottom: 50px;
```

Comment:
a block can't have margins
use mix to fix this (by making each card an element of cards block)
https://www.youtube.com/watch?v=JOJ-mENrg_s